pkg/kubelet/util/util_windows.go

```diff
@@ -55,3 +56,15 @@ func GetAddressAndDialer(endpoint string) (string, func(addr string, timeout tim
 func dial(addr string, timeout time.Duration) (net.Conn, error) {
 	return net.DialTimeout(tcpProtocol, addr, timeout)
 }
+
+var tickCount = syscall.NewLazyDLL("kernel32.dll").NewProc("GetTickCount64")
+
+// GetBootTime returns the time at which the machine was started, truncated to the nearest second
+func GetBootTime() (time.Time, error) {
+	currentTime := time.Now()
+	output, _, err := tickCount.Call()
+	if errno, ok := err.(syscall.Errno); !ok || errno != 0 {
+		return currentTime, err
```

Comment:
Why returning the currentTime as the boot time on error?

Reply:
Discussed offline.  Returning time.Time{} (the zero time) seems like the most obvious way to fail here.